src/zscale.jl

```diff
@@ -0,0 +1,248 @@
+MAX_REJECT = 0.5
+MIN_NPIXELS = 5
+GOOD_PIXEL = 0
+BAD_PIXEL = 1
+KREJ = 2.5
+MAX_ITERATIONS = 5
```

Comment:
Please, have a read of the performance tips in the Julia manual: https://docs.julialang.org/en/v1/manual/performance-tips/.  **Not** using non-constant global variables is the very first issue.  Some of these variables aren't even used in more than a function, so making them global isn't even useful.